.cirrus.yml

```diff
@@ -81,10 +81,10 @@ test_task:
     matrix:
         - name: "Test"
           env:
-              BUILDTAGS: 'btrfs_noversion'
+              BUILDTAGS: ''
```

Comment:
https://github.com/containers/skopeo/pull/2581 needs to be merged first, for the “Don't require BUILDTAGS to be non-empty” commit.